Explorer/Assets/Scripts/Global/Dynamic/RealmController.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using Arch.Core;
 using CommunicationData.URLHelpers;
 using Cysharp.Threading.Tasks;
+using DCL.CommunicationData.URLHelpers;
```

Comment:
Probably this import is not needed since there is no code usage of this here right?

Reply:
It is used so i can check if the provided url is an ens name